CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
I would not change this unless it is necessary for some reason. Same comment for the other cmake files.

Reply:
Cool. Will undo.